### ISSUE TITLE
Design "pre/during MP" state for /events

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.webmaker.org#843]
This needs more grooming, and is blocked on the creation of the MP activities and case studies, but the /events page will need some adjusting during the lead up to Maker Party

### Requirements ###
* Prominently feature an "I want to host a Maker Party!" workflow (user enters email address, transactional email is sent with links to three MP activities, event resources, and certificate template) - this is basically a re-purposing of the "Sign up" flow currently on the page
* Prominently feature the official MP activities (see https://github.com/mozilla/learning-networks/issues/169)
* Showcase the MP case studies (see https://github.com/mozilla/learning-networks/issues/137)

**Comment by hannahkane** [comment]
TDB: 
* Any copy changes?
* Remove video and/or photos?
* Do we want to remove the link to /resources (from both nav and bottom of page), and just email the link to people who sign up?

**Comment by toolness** [comment]
Did you mean TBD or is TDB an acronym that I'm unaware of?

**Comment by hannahkane** [comment]
Corrected. Thanks!

**Comment by ldecoursy** [comment]
Few updates:

* @amirad drafting updated copy for the /events page which we'll have by Friday (see https://github.com/mozilla/learning-networks/issues/175)
* Let's keep the video, but remove the current Flickr photos - we'll have photos for the case studies anyway. Would still be good to keep a link to the Flickr feed if that works.
* @amirad wants to keep the link to event resources 
* Final case studies are in https://github.com/mozilla/learning-networks/issues/137

**Comment by ldecoursy** [comment]
Here's the link to the copy/flow/links on the events page: https://teach.etherpad.mozilla.org/teachsite-eventspage

**Comment by ldecoursy** [comment]
@HPaulJohnson Can we get copy review on the link above? This is new copy for the /events page in prep for Maker Party.

**Comment by sabrinang** [comment]
WIP - arranging content and copy tbd
![events page inital mp mockup](https://cloud.githubusercontent.com/assets/2492510/7848124/c2a79b96-047d-11e5-8b25-e30f943a3717.png)

**Comment by hannahkane** [comment]
Thanks, @sabrinang, for sharing the WIP. You've done a great job already, but it's helping me to see that this page is bonkers. :) We're trying to put too much stuff on one page. 

My recommendations:
 * The two consecutive blocks of text under the hero are too much. Suggest removing the first one entirely.
 * Between now and the end of the campaign, the primary CTA needs to be "I'll host a Maker Party" rather than "Sign up to receive updates." The user interaction is the same—the user submits their email address and receives an automatic email (copy changes for the email TK).
 * Though I like the look of the primary CTA in the hero image, it should probably be connected to the "Host a Maker Party..." paragraph.
 *  Suggest adding a simple, short description of what a Maker Party is as the sub-head in the hero. e.g. "Get together with others to learn about the web by making things." (def open to better copy there. I'm kind of blanking on the clearest, shortest way to describe it)
 * I'd recommend removing the "year-round" language for between now and July 31st. After July 31st, we can put in all the "year-round" messaging we want, but until then, I think it's confusing and a missed opportunity to maximize the two-week campaign. We want to create a sense of urgency for the campaign.
 * Suggest we put the activities on their own page, and link to it prominently from this page (can use existing activity page as the template - and hey, we could even link to that page from the /activities page under a heading, "Maker Party 2015 Recommended Activities", or something like that)
 * Suggest removing the links to /activities and /clubs pages. They are distracting on this page. Plus, the Clubs link is an upsell that will happen via email *after* the campaign.
 * Suggest removing "Watch this video or explore our photo gallery" sentence, since both of those things already appear elsewhere on the page
* Suggested copy change: "Want to know more about Maker Party?" could be "What does a Maker Party look like?" with paragraph copy "Maker Parties are held in schools, cafes, community spaces, or even around kitchen tables. They range from the very large (hundreds of participants) to the very small (two people). They are great for kids and adults, and for beginners or experienced pros. Check out these examples of fantastic Maker Parties."
 * OMG, who participated in the Lo-fi Maker Party??? :)
 * Shouldn't repeat photo from case study in Flickr gallery

@ldecoursy @amirad @HPaulJohnson - thoughts?

**Comment by ldecoursy** [comment]
Agree with your comments above. 

* Would it make sense to have "Read, Write, Participate" activities laid out horizontally on the page? I don't think the lo-fi alternatives need thumbnails or descriptions. 
* Per a previous comment in this ticket, we can lose the three flickr thumbnails at the bottom of the page, since we have the case studies. 
* Can we just have two case studies instead of three?

**Comment by amirad** [comment]
Agree with comments above.
•Do we need flickr roll? Can we just say see more event photo's in our flickr gallery since we are already showing pictures with case studies? I had assumed that's what we were doing. 
•Agree to lay out activities horizontally and not have thumbnails for the lo-fi options. We can make that section a lot tighter (i.e. no read/write/participate banners either. Just three images and text under). 
•Should we shorten text for case studies? They look big and clunky right now. I like all three since they show a good spread of locations, event types and people.

**Comment by hannahkane** [comment]
I feel like there might be some key advantages to putting the activities on a separate page (using the existing template for activity pages - see Web Lit Basics) and prominently linking to it from this page. 1) we'd have a specific page to link to in the follow-up email, 2) they're good activities! We should link to them from the /activities page, too., and 3) gives this page some breathing room.

Not 100% on that, though. If others thing they need to be on this page, then I support Amira's suggestion for tightening it up.

+1 to keeping the three case studies. I think three is a good number—probably worth playing around with additional layout options to accommodate all three.

**Comment by amirad** [comment]
@HPaulJohnson @KevZawacki and I just met and think having them on their own dedicated page would help for marketing purposes.

**Comment by HPaulJohnson** [comment]
A little late to this but +1 to all of Hannah's suggestions above. As @amirad mentioned I think we can eliminate much of the information density by moving activities to dedicated pages and it will help when we want to link to specific ones over the course of our campaign communications. 

For main CTA we should put the urgency front and center. I suggest: 
Host a Maker Party
Join the global celebration from July 15-31 and create awesome things on the open Web. 
[Get Started] or [Join] or [I'll Host a Party] or [Count Me In!] 

By putting it in the headline, I don't know that it needs to be repeated again directly below. Would it work to have the "Join the Global Movement"  paragraph take up it's own horizontal space so that the font could be bigger, put the video directly below and then "Get Started" with brief descriptions of read, write, participate that link of to individual activity pages?

Lastly, I might repeat the CTA again, either after activities or after "want to know more." If people dig that deep, give them another easy way to join below the fold.

**Comment by sabrinang** [comment]
![events page mockup 02062015](https://cloud.githubusercontent.com/assets/2492510/7948906/5be8c398-093d-11e5-90a9-5dbb52fc4014.png)
Missing copy for "Get Started" with brief descriptions of read, write, participate that link of to individual activity pages or if this is going to be "Maker Party 2015 Recommended Activities"

**Comment by hannahkane** [comment]
My suggestions:
 * I think the button copy in the hero needs to make it clear that they're signing up to host. How about, "COUNT ME IN"?
 * I don't think we need individual pages for Read, Write, and Participate. I think we can have a single page that has *all* of the Maker Party activities. So, I think it should be a prominent link or button that reads "Get the 2015 Maker Party Activities"
 * Copy change on sign-up at bottom. Replace "Want to know more? Sign up to get Maker Party updates:" with "Ready to get started? Sign up to host a Maker Party:"

**Comment by ldecoursy** [comment]
Do we need both sign-up's? I'd suggest just keeping the one at the top. Maybe the sub head says "Sign up to join the global celebration..."

**Comment by hannahkane** [comment]
@ldecoursy - can you include the full copy you're suggesting? Do you mean, "Sign up to join the global celebration from July 15-31 and create awesome things on the open Web"? I feel like that copy is already too long. Would love to make it one line. "Join the global celebration from July 15-31." And maybe the button is "SIGN UP".

I vote for keeping two CTAs on the page, though. The page does the work of convincing someone to sign up, so I think we need to reiterate the CTA at the bottom.

Just noticed a typo. Should read: "See more event photos in our Flickr gallery" (no apostophe in photos)

**Comment by ldecoursy** [comment]
Agree on shortening the copy - so that's the same as the copy on the MP promo on the homepage. 

I guess if we keep both CTAs, it should just be clear from the language that they're the same thing so people don't need to submit their email twice - essentially they're going to the same list.

**Comment by ldecoursy** [comment]
@hannahkane @sabrinang Just checking to see if we're set for copy on this page. If so, I'll close https://github.com/mozilla/learning-networks/issues/175 :)

**Comment by hannahkane** [comment]
I think we can make these copy changes to the most recent version:
 * Header: "Join the global celebration from July 15-31." Button: "SIGN UP" (can we put the email address field in the header, so it looks similar to the CTA on the bottom?)
 * Link to activities: Prominent link or button that reads "Get the 2015 Maker Party Activities"
 * Fix typo: "See more event photos in our Flickr gallery" (no apostophe in photos)
 * CTA on bottom: "Ready to host a Maker Party?" Button after email address field: "SIGN UP"

**Comment by sabrinang** [comment]
WIP (will continue iterating on the _Link to activities_)
![events page mp iteration](https://cloud.githubusercontent.com/assets/2492510/8014391/3511b4ba-0b85-11e5-83c7-268dced79493.png)

**Comment by hannahkane** [comment]
I am really liking this. Will file a separate ticket to create the MP Activities page that this will link to.

**Comment by amirad** [comment]
+1 it's looking good!

**Comment by sabrinang** [comment]
added some MP illustrations near the Activities button
![events page - activities cta](https://cloud.githubusercontent.com/assets/2492510/8072150/d2ecfbec-0ec6-11e5-80fc-04513de00e6a.png)
redpen: https://redpen.io/p/zoe6d4942cfdef74d2
@cassiemc R?

**Comment by cassiemc** [comment]
Hey @sabrinang, I haven't been too involved in this process but I left you some feedback in the redpen. In general the layout of the page is good, but some of the elements need finessing to make sure they really fit the design of the page. Please ping me if you want to go over anything by voice, and it'd be great if you could get @flukeout to offer to some feedback here too.

**Comment by sabrinang** [comment]
updated redpen: https://redpen.io/p/zoe6d4942cfdef74d2
![events page - iteration4 - 061515](https://cloud.githubusercontent.com/assets/2492510/8172494/ed49f02c-1375-11e5-844d-4f8f9c3fabbd.png)

**Comment by hannahkane** [comment]
Still a +1 from me. Need an R from @cassiemc?

**Comment by cassiemc** [comment]
R+ we can go ahead with this, perhaps iterating on the header banner in the future!

**Comment by hannahkane** [comment]
w00t. Thanks, @cassiemc. @sabrinang - can you please put any assets on the implementation ticket? #900
